### PR TITLE
init0

### PR DESCRIPTION
**Pull request by hemanth**
@mklabs Just some base code to start with.

### CODE REVIEW

**Comment by mklabs** [comment]
Awesome!

Merging!

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const bindings = require('./bindings.json');
+const keymaster = require('keymaster');
+
+module.exports = function (str, opts) {
+	const bindingKeys = Object.keys(bindings);
```
Comment (mklabs) [inline index.js:7]: what's your spacing config ?

**Comment by mklabs** [comment]
Great stuff, thanks a lot @hemanth 

We'll init eslint / babel setup, you can use https://github.com/mklabs/bake#bake-init with [default template](https://github.com/mklabs/bake/tree/master/templates/default) if you wish to do it and automate that.

I still need an hour or two before getting into nmap.

**Comment by hemanth** [comment]
Sure, I shall use `bake-init`

File: index.js (hunk 1 above)
Comment (hemanth) [inline index.js:7]: It must pick up form the `.editorconfig` actually.

**Comment by mklabs** [comment]
Let me know how it works on your OS / setup.

I'm the only user for now. Might break.

**Comment by hemanth** [comment]
```
$ bake-init 
bake ℹ info Running default template in  /Users/hhegadehallimadh/labs/nmap
bake ℹ info prestart script
prestart
bake ℹ info start script
Starting generation of default template
bake ℹ info poststart script
poststart
bake ⚠ warn package.json  already exists, merging
bake ℹ info .npmignore    -> ./.npmignore
bake ℹ info package.json  -> ./package.json
bake ℹ info .eslintignore -> ./.eslintignore
bake ℹ info Makefile      -> ./Makefile
bake ℹ info preinstall script
Installing dependencies ...
bake ℹ info install script
npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/hhegadehallimadh/.nvm/versions/node/v6.0.0/bin/node" "/Users/hhegadehallimadh/.nvm/versions/node/v6.0.0/bin/npm" "install" "--loglevel" "warn" "--cache-min" "Infinity"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.6

npm ERR! No compatible version found: eslint-config-standard@^5.3.1
npm ERR! Valid install targets:
npm ERR! 5.2.0, 5.1.0, 5.0.0, 4.4.0, 4.3.2, 4.3.1, 4.3.0, 4.2.0, 4.1.0, 4.0.0, 4.0.0-5, 4.0.0-4, 4.0.0-3, 4.0.0-2, 4.0.0-1, 4.0.0-0, 3.4.1, 3.4.0, 3.3.0, 3.2.0, 3.1.0, 3.0.0, 2.0.2, 2.0.1, 2.0.0, 1.3.1, 1.3.0, 1.2.0, 1.1.1, 1.1.0, 1.0.1, 1.0.0
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/hhegadehallimadh/labs/nmap/npm-debug.log
bake ✖ ERR  npm install --loglevel warn --cache-min Infinity
bake ✖ ERR  Error: Recipe exited with code %d
    at ChildProcess.<anonymous> (/Users/hhegadehallimadh/.nvm/versions/node/v6.0.0/lib/node_modules/bake-cli/src/cli.js:143:28)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
bake ℹ info Build success in 4395ms
```

On node `v6.0.0`

**Comment by mklabs** [comment]
woot

**Comment by mklabs** [comment]
new, i tested the template on windows 10 and arch, but it looks like its just a package issue.

**Comment by hemanth** [comment]
heh heh yeah, I was so lazy and just paste the issue.

**Comment by mklabs** [comment]
automatic colored output for text syntax higlight, begining with a $ would be awesome.

**Comment by mklabs** [comment]
like on travis, there's a package that does the conversion between ansi code and html class name.

**Comment by hemanth** [comment]
yeah, that would make things look neater and life easier!

**Comment by hemanth** [comment]
Fixing `eslint-config-standard@^5.3.1` results in `No compatible version found: module-deps@^4.0.6`

**Pull request merged**